README.md

```diff
@@ -1 +1,29 @@
-# Operator Marketplace
+# Marketplace Operator
+
+## Project Status: pre-alpha
+
+## Prerequisites
+In order to deploy the Marketplace Operator, you must:
+1. Have an OpenShift cluster with Operator Lifecycle Manager [installed](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/install/install.md).
+2. Be logged in as a user with the RBAC permissions required to deploy this operator. These RBAC permissions are currently undefined, so log in as a user with the Cluster Admin Role.
+
+## Making changes to the Marketplace Operator
+The Marketplace Operator is hosted at `quay.io/redhat/marketplace-operator` and not all developers have pull and push privileges on this image. If you do not have these privileges and are developing new features for the Marketplace Operator you must build and push your image to a registry where you have said privileges. The steps below outline said process:
+1. Build and push your Marketplace Operator Image with the following command.
+```bash
+$ export REGISTRY=<SOME_REGISTRY> \
+   && export NAMESPACE=<SOME_NAMESPACE> \
+   && export REPOSITORY=<SOME_REPOSITORY> \
+   && export TAG=<SOME_TAG> \
+   && operator-sdk build $REGISTRY/$NAMESPACE/$REPOSITORY:$TAG \
+   && docker push $REGISTRY/$NAMESPACE/$REPOSITORY:$TAG
+```
+2. Update the `deploy/operator.yaml` to pull the Marketplace Operator image you just pushed. You should update the `spec.template.spec.containers[0].image` field with the `$REGISTRY/$NAMESPACE/$REPOSITORY:$TAG` value.
+
+You can now deploy your operator by following the steps outlined in the [Deploying the Marketplace Operator](#deploying-the-marketplace-operator) section.
```

Comment:
Just my opinion, but this line and link are a bit redundant since the Deploy section is right below.

Reply:
Removed the line pointing to the "Deploying the marketplace operator" section.